Vizzuality/widget-editor
# filter selection bug fixes

Description of the PR

This pr fixes multiple issues related to filter column selection. 

1. as the pivotal task reported there is an edge case where the select options stay open. This happens when you select a column and click right away before everything is loaded. Now I'm disabling it locally so that's not possible. 
2. Improved loading state for filters by supplying a saga that checks when they are loading. This is applied to the selector as well to avoid selection when loading. 
3. Fixed bug causing long values to overflow deletion box within select. 

### Bugs

<img width="579" alt="Screenshot 2020-10-29 at 11 48 12" src="https://user-images.githubusercontent.com/971129/97558972-268a5980-19dd-11eb-9660-81f0655145e7.png">

<img width="519" alt="Screenshot 2020-10-29 at 11 39 17" src="https://user-images.githubusercontent.com/971129/97559019-3144ee80-19dd-11eb-987f-bd350ccaca4e.png">

## Testing instructions

1. Make sure when making a selection and quickly making another one that the options don't stay open when selecting a column. 

## Pivotal Tracker

[task](https://www.pivotaltracker.com/story/show/175279372)
